build/Tests/UnitTests/CoreGraphics/CoreGraphics.UnitTests.vcxproj

```diff
@@ -239,6 +239,11 @@
     <ClangCompile Include="..\..\..\..\tests\unittests\CoreGraphics\CGContextTests.mm" />
     <ClangCompile Include="..\..\..\..\tests\unittests\CoreGraphics\CGBitmapContextTests.mm" />
     <ClangCompile Include="..\..\..\..\tests\unittests\CoreGraphics\CGColorTests.mm" />
+    <ClangCompile Include="..\..\..\..\tests\unittests\CoreGraphics\CGImageTests.mm" />
+    <ClangCompile Include="..\..\..\..\tests\unittests\CoreGraphics\TestUtils.mm" />
+  </ItemGroup>
+  <ItemGroup>
+    <SBResourceCopy Include="..\..\..\..\tests\unittests\CoreGraphics\images\**\*" />
```

Comment:
changed to copy after build and not be part of the SDK

---
In reply to: [84752712](https://github.com/Microsoft/WinObjC/pull/1185#discussion_r84752712) [](ancestors = 84752712)